core/src/main/java/hudson/cli/InstallPluginCommand.java

```diff
@@ -113,8 +115,9 @@ protected int run() throws Exception {
                     pm.dynamicLoad(f);
                 }
                 continue;
-            } catch (MalformedURLException e) {
+            } catch (MalformedURLException | URISyntaxException e) {
                 // not an URL
+                e.getCause();
```

Comment:
this is dead code - you did not need to add this and spotbugs is probably complaining 

```suggestion
```